lib/typinggame_server/interactors/players/index.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'hanami/interactor'
+
+module Interactors
+  module Players
+    class Index
```

Comment:
This isn't my favourite name, as it's very closely tied to the implementation on the web side.

I would favour something more along the lines of `FetchAllPlayers`. We want files in our lib directly to be able to stand alone. We don't care that this is being used in the context of an index action.

Reply:
@jgayfer What about the file name? `fetch_all_players.rb` ?

Reply:
@jgayfer fixed